client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -114,11 +122,25 @@ const selectedLabel: ComputedRef<string> = computed(() => {
  */
 const selectedValues = computed(() => (Array.isArray(props.value) ? props.value : [props.value]));
 
+/**
+ * Tracks selected ids in case of objects with string ids
+ */
+const selectedIds = computed(() => {
+    return selectedValues.value.map((v) => (isValueWithId(v) ? v.id : undefined)).filter((v) => v !== undefined);
```

Comment:
What are those ids ? dataset ids ? Those are not unique, you'd typically have to combine this with `src` or `history_content_type`